packages/cli-kit/src/git.ts

```diff
@@ -30,12 +31,38 @@ export const DetachedHeadError = () => {
   )
 }
 
+export const NothingToCommitError = () => {
+  return new Abort('Nothing to commit')
+}
+
 export async function initializeRepository(directory: string) {
   debug(content`Initializing git repository at ${token.path(directory)}...`)
   await ensurePresentOrAbort()
   await git(directory).init()
 }
 
+export async function createGitIngore(directory: string) {
+  debug(content`Creating .gitignore at ${token.path(directory)}...`)
```

Comment:
Is this for our own purposes?

Reply:
Just copying what was already there in other functions like `initializeRepository`.